app/models/redirect.rb

```diff
@@ -39,6 +40,9 @@ def create_redirect_in_publishing_api
     throw :abort # Do not continue to save
   end
 
+  def unpublish_in_publishing_api
+    GdsApi.publishing_api_v2.unpublish(content_id, type: "gone")
```

Comment:
Do we want to abort if this fails? Is it possible that we could end up with redirects still present in router-api but not in short-url-manager?

Reply:
Will the abort happen automatically if it fails (and raises an exception) or do we need to call `throw :abort`? I'll add a test...

Reply:
If it raises an exception it will rollback. Would this always raise an exception if it fails?

https://guides.rubyonrails.org/v2.3.11/activerecord_validations_callbacks.html#halting-execution

Reply:
It'll raise an exception if the HTTP request returns an error status code, so I think we're good. I've added a test for this situation as well.